[2025-05-05T04:04:03.704+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-05-05T04:03:00+00:00 [queued]>
[2025-05-05T04:04:03.712+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-05-05T04:03:00+00:00 [queued]>
[2025-05-05T04:04:03.712+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-05T04:04:03.728+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): load_mysql_data> on 2025-05-05 04:03:00+00:00
[2025-05-05T04:04:03.733+0000] {standard_task_runner.py:60} INFO - Started process 15022 to run task
[2025-05-05T04:04:03.737+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mymind_mongo_to_mysql_etl', 'load_mysql_data', 'scheduled__2025-05-05T04:03:00+00:00', '--job-id', '28826', '--raw', '--subdir', 'DAGS_FOLDER/mymind_etl_dag.py', '--cfg-path', '/tmp/tmpy8jr03tw']
[2025-05-05T04:04:03.738+0000] {standard_task_runner.py:88} INFO - Job 28826: Subtask load_mysql_data
[2025-05-05T04:04:03.787+0000] {task_command.py:423} INFO - Running <TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-05-05T04:03:00+00:00 [running]> on host 9548ec498b56
[2025-05-05T04:04:03.897+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mymind_mongo_to_mysql_etl' AIRFLOW_CTX_TASK_ID='load_mysql_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-05T04:03:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-05T04:03:00+00:00'
[2025-05-05T04:04:03.899+0000] {mymind_etl_dag.py:285} INFO - Iniciando carga a MySQL: 4 usuarios, 15 transcripciones.
[2025-05-05T04:04:03.909+0000] {base.py:83} INFO - Using connection ID 'mysql_mymind_dw' for task execution.
[2025-05-05T04:04:03.921+0000] {mymind_etl_dag.py:304} INFO - Cargando 4 usuarios en la tabla users...
[2025-05-05T04:04:03.922+0000] {mymind_etl_dag.py:329} INFO - Usuarios cargados exitosamente.
[2025-05-05T04:04:03.922+0000] {mymind_etl_dag.py:342} INFO - Cargando 15 transcripciones en la tabla transcriptions...
[2025-05-05T04:04:03.925+0000] {mymind_etl_dag.py:372} INFO - Transcripciones cargadas exitosamente.
[2025-05-05T04:04:03.926+0000] {mymind_etl_dag.py:379} INFO - Carga a MySQL completada con Ã©xito.
[2025-05-05T04:04:03.926+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-05T04:04:03.937+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mymind_mongo_to_mysql_etl, task_id=load_mysql_data, execution_date=20250505T040300, start_date=20250505T040403, end_date=20250505T040403
[2025-05-05T04:04:03.989+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-05T04:04:04.021+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
